### DISCUSSION TITLE
OneDrive Client Version v2.5.0 Feedback Request

### DISCUSSION THREAD

**Discussion by abraunegg** [abraunegg/onedrive#2809] [General]
With v2.5.0 released, I would like to start gathering feedback on the release given the significant amount of change between v2.4.25 and v2.5.0.

If you have upgraded and using v2.5.0 please post your feedback (good or bad) here.

Thanks in advance.

**Comment by kayzzen01** [comment]
Hello,

I had a small problem with libcurl, because the version I was using had a bug related to the HTTP version error.
I struggled a bit, but I already had the answer in the documentation and the solution in pacman -Syu.

So far, the monitor mode is cleaner, avoiding OneNote files.
The application has an excellent memory consumption of 36.6MB :grinning:.

I have a question and I don't know if it's the best place to ask this. It seems that now OneDrive's behavior is to queue the inotify changes and after 5 minutes process the local changes instead of processing them in real time. Would this really be the expected behavior?

**Reply by abraunegg** [reply]
> I have a question and I don't know if it's the best place to ask this. It seems that now OneDrive's behavior is to queue the inotify changes and after 5 minutes process the local changes instead of processing them in real time. Would this really be the expected behavior?

inotify changes are queued for ~5 seconds then processed in a batch. Online changes are checked every 5 minutes based on 'monitor_interval'

That is the expected behavior and what has been validated | tested | confirmed as what occurs.

Now .. big caveat - and *this is* called out in the documentation: https://github.com/abraunegg/onedrive/blob/master/docs/client-architecture.md#what-are-the-operational-modes-of-the-client

![image](https://github.com/user-attachments/assets/2d444ae3-692f-4c82-acb1-cd7140e19a7a)

If your OneDrive data is on a NAS or an external network mount point (such as NFS, Windows Network Share, or Samba Network Share) - SMB and NFS lack inotify support. This is nothing this project can fix, thus, it would *appear* to you that local inotify changes are only occurring every 5 minutes due to the 'monitor_interval' value.

**Reply by kayzzen01** [reply]
My files are kept locally.

So there must be some problem with my OneDrive Client configuration or my operating system. Both local changes and online changes are being processed in the 'monitor_interval' time. I'll open an issue to try to figure out the problem.

**Reply by abraunegg** [reply]
Let me look into this .. as .. there have been so *odd* regressions of items / things that have been tested ...

**Reply by abraunegg** [reply]
Confirmed - yet another regression .... please raise a bug

**Reply by abraunegg** [reply]
OK .. this is ONLY a regression if there is a 'sync_list' in use, then the a new local file does not get uploaded ... which is odd ....... and is certainly a regression.

**Reply by kayzzen01** [reply]
OK. An issue has been opened https://github.com/abraunegg/onedrive/issues/2832.
Thanks.
